tasks/publish-to-mrrc/publish-to-mrrc.yaml

```diff
@@ -42,27 +49,33 @@ spec:
       script: |
         #!/usr/bin/env bash
         set -eux
+
+        CHARON_CFG_FILE="$(workspaces.data.path)/$(params.charonConfigFilePath)"
+        mkdir -p "/home/charon/.charon"
```

Comment:
Have you tested this? I am not sure `/home` is typically writable in tekton tasks

Reply:
I've tested this in my personal workspace and it is working. There is a test assertion in the tests to make sure of this. If the managed workspace is working in same way I think it should work.

Reply:
What do you mean in your personal workspace? In a pipelineRun in your workspace in konflux?

Reply:
Yes. You can see it here: https://github.com/ligangty/konflux-mrrc-release/blob/main/release/pipelines/mrrc-release-pipeline.yaml. 
And if no exception this one will be the final pipeline for the whole work.